src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int LIMIT_OF_NUMBERS = 101;
+    private Random randomValuesSupplier = new Random();
+
+    public Ball getRandomBall() {
+        int randomInt = randomValuesSupplier.nextInt(LIMIT_OF_NUMBERS);
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
You have Random as a class field and ColorSupplier local in the method. What is the difference between these two objects? Why did you decide to do it this way?

Reply:
I made changes to Main() method and Lottery class.